ssl/ssl_lib.c

```diff
@@ -454,44 +454,44 @@ static int ssl_check_allowed_versions(int min_version, int max_version)
             return 0;
     } else {
         /* Regular TLS version checks. */
-	if (min_version == 0)
-	    min_version = SSL3_VERSION;
-	if (max_version == 0)
-	    max_version = TLS1_3_VERSION;
+    if (min_version == 0)
```

Comment:
Starting here I think we need another 4 spaces of indentation.

Reply:
Ok, I'll have a closer look at the code. Originally, I was working on something else and just did a 'whitespace-cleanup' in Emacs, when I noticed these extra changes. So to keep them out of my PR, I created a new one.